src/App.tsx

```diff
@@ -15,49 +19,53 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const getSortedGoods: GetFuncType = (goods, { sort, reverse }) => {
+  if (sort) {
+    goods.sort((a: string, b: string) => {
+      switch (sort) {
+        case sortBy.alphabet:
+          return a.localeCompare(b);
+
+        case sortBy.length:
+          return a.length - b.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    return goods.reverse();
+  }
+
+  return goods;
+};
+
 export const App: React.FC = () => {
+  const [sortedBy, setSortedBy] = useState<string>('');
+  const [reverse, setReverse] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isReversed, seIstReverse] = useState<boolean>(false);
```
Boolean variables should have the prefix `is` or `has`
And actually, you don't have to specify generic when you pass a default value to useState